.github/workflows/api_tests.yaml

```
@@ -14,8 +14,17 @@ jobs:
     name: Run API Test AKME
     runs-on: ubuntu-latest
     steps:
+    - name: Generate GitHub App token
+      id: app-token
+      uses: tibdex/github-app-token@v2
+      with:
+        app_id: ${{ secrets.REPO_READ_WRITE_APP_ID }}
+        private_key: ${{ secrets.REPO_READ_WRITE_APP_PRIVATE_KEY }}
+
     - name: Checkout repository
       uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)